src/fireflyClient.ts

```diff
@@ -897,8 +897,15 @@ export class FireflyClient {
    */
   private createOrderToSign(params: OrderSignatureRequest): Order {
     const expiration = new Date();
```

Comment:
I think in node Date returns unix timestamp in milliseconds. On contract level, the smallest unit of time is in seconds. 
Whatever expiration time you are getting you need to divide it by 1000 to convert it into unix time stamp in seconds.
Something like this has to be done to make market order expirable in 1 min: `Math.floor(Date.now()/1000) + 60`

Reply:
Yes /1000 was added before but dapi expiration was changed to miliseconds that's why i removed /1000. will add it back when it's fixed on dapi.

Reply:
If i /1000 dapi gives me error "Expiration time must be in the future." Abdullah has fixed this thing on dapi once his PR gets merged i'll make changes here

Reply:
We should either merge this PR with the 1000 division factor or not. Let's wait for DAPI to be fixed then.

Reply:
PR is merged on dapi and fixed here @YameenMalik